op-challenger/fault/types/types.go

```diff
@@ -26,12 +27,44 @@ type StepCallData struct {
 	Proof      []byte
 }
 
+// PreimageOracleData encapsulates the preimage oracle data
+// to load into the onchain oracle.
+type PreimageOracleData struct {
```

Comment:
Seems like maybe we should use this struct everywhere we're tracking the preimage key and data? It's usually better to be consistent with the type we use for something rather than sometimes having it wrapped and sometimes not.